aws/sqs_event_manager/README.rst

```diff
@@ -1,6 +1,83 @@
-AWS SQS interface
+AWS SQS Event Manager
+---------------------
 
-React on SQS events connected as lambdas. 
+The SQS Event Manager reacts on SQS events from a given sns topic.
+The topic of the messages are marketplace entitlement change notifications.
+The manager runs as a Lambda function and handles the events when the come
+in. The cusomer entitlement data is collected and passed on to the
+API setup in the coniguration file.
+
+Higlevel Deployment Plan
+------------------------
+
+To deploy the sqs_event_manager code in the AWS cloud the
+following services must be used:
+
+* AWS Elastic Container Registry
+* AWS Lambda
```

Comment:
I believe creating the SQS queue also belongs to the deployment, no ?

Reply:
Yes, SQS  and SNS are separate services that should be listed here.

Reply:
We should probably in a generic way mention that routing and network need to be configured to appropriately route the call to the configured API.